src/ngx-gallery-scroll-overview/ngx-gallery-scroll-overview.component.ts

```diff
@@ -69,8 +66,9 @@ export class NgxGalleryScrollOverviewComponent implements AfterViewInit {
 
     onStop(event) {
         var rect = event.getBoundingClientRect();
-        this.gallery.positionLeft += (this.beforeZoomLeft - rect.left) * this.SCALE_FACTOR;
-        this.gallery.positionTop += (this.beforeZoomTop - rect.top) * this.SCALE_FACTOR;
+        this.zoomPosition.positionLeft += (this.beforeZoomLeft - rect.left) * this.SCALE_FACTOR;
+        this.zoomPosition.positionTop += (this.beforeZoomTop - rect.top) * this.SCALE_FACTOR;
+        this.onZoomChanged.emit();
```

Comment:
And here you would emit the before and after zoom position: `this.zoomChanged.emit(zoomPositions)` (you have to build this object before of course).